ios/FluentUI/Pill Button Bar/PillButtonBar.swift

```diff
@@ -183,7 +183,11 @@ open class PillButtonBar: UIScrollView, TokenizedControlInternal {
 
     @objc public let pillButtonStyle: PillButtonStyle
 
-    @objc public var pillButtonOverrideTokens: PillButtonTokens?
+    @objc public var pillButtonOverrideTokens: PillButtonTokens? {
+        didSet {
+            updatePillButtonTokens()
```

Comment:
It looks like the pill button bar doesn't respond to `didChangeTheme` notifications, so it stays with whatever theme it was presented with. Is this known/tracked?

https://user-images.githubusercontent.com/4934719/157120197-501a6319-b0e6-4edf-8305-52a91d9c8371.mp4

Reply:
I don't think there's anything tracking that, but I can make a bug.